plugins/alt-text/index.js

```diff
@@ -18,6 +18,7 @@ class AltTextPlugin extends Plugin {
 
     reportError(el) {
         let $el = $(el);
+        let src = $el.prop("src") || "..";
```

Comment:
If we change the failing image in `test.html` to have a *relative* src (i.e. `src="jdan.png"`), we see the following:

![image](https://cloud.githubusercontent.com/assets/287268/9859508/f50aedb8-5af4-11e5-8f10-6d9fa3658341.png)

This is somewhat problematic as it does not accurately reflect the contents of the `src` attribute. The reason for this is because jQuery's `.prop()` will [resolve any URLs using the following process](http://www.w3.org/TR/html5/infrastructure.html#reflecting-content-attributes-in-idl-attributes). 

`.attr()` will return the actual contents of the `src` attribute, can we use that instead? You can even do `el.getAttribute("src")` and skip jQuery all together.